server/src/test/java/io/spine/server/command/model/AssigneeReceptorTest.java

```diff
@@ -72,10 +72,11 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 @DisplayName("`CommandAssigneeMethod` should")
```

Comment:
And here. 

I will not double this comment anymore. It looks that display names were not addressed when renaming.